Lacks a necessary import

Following error happens
`Assets/WalletConnectUnity/Demo/Full Session/Scripts/DemoActions.cs(53,9): error CS0246: The type or namespace name 'List<>' could not be found (are you missing a using directive or an assembly reference?)`

It can be fixed by adding an import to DemoActions.cs
```
using System.Collections;
```

https://github.com/WalletConnect/WalletConnectUnity/pull/55

--- a/Assets/WalletConnectUnity/Demo/Full Session/Scripts/DemoActions.cs
+++ b/Assets/WalletConnectUnity/Demo/Full Session/Scripts/DemoActions.cs
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Collections;
 using UnityEngine;
 using UnityEngine.UI;
 using WalletConnectSharp.Core;

